doc/classes/ProjectSettings.xml

```diff
@@ -2364,6 +2364,136 @@
+			The maximum relative angle by which a body pair can move and still reuse the collision results from the previous physics step.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/body_pair_contact_cache_distance_threshold" type="float" setter="" getter="" default="0.001">
+			The maximum relative distance by which a body pair can move and still reuse the collision results from the previous physics step.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/body_pair_contact_cache_enabled" type="bool" setter="" getter="" default="true">
+			Whether the body pair contact cache is enabled, which removes the need for potentially expensive collision detection when the relative orientation between two bodies hasn't changed much.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/bounce_velocity_threshold" type="float" setter="" getter="" default="1.0">
+			The minimum velocity needed before a collision can be elastic.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/continuous_cd_max_penetration" type="float" setter="" getter="" default="0.25">
+			Fraction of its inner radius a body may penetrate another body whilst using CCD.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/continuous_cd_movement_threshold" type="float" setter="" getter="" default="0.75">
+			Fraction of its inner radius a body must move per step to make use of CCD.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/generate_all_kinematic_contacts" type="bool" setter="" getter="" default="false">
+			Whether or not a [RigidBody3D] frozen with [constant RigidBody3D.FREEZE_MODE_KINEMATIC] is able to collide with (and thus generate contacts for) other kinematic (and static) bodies.
+			[b]Note:[/b] This setting can come at a heavy CPU and memory cost if you allow many/large frozen kinematic bodies with a non-zero [member RigidBody3D.max_contacts_reported] to overlap with complex static geometry, such as [ConcavePolygonShape3D] or [HeightMapShape3D].
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/penetration_slop" type="float" setter="" getter="" default="0.02">
+			How much bodies are allowed to penetrate each other.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/position_steps" type="int" setter="" getter="" default="2">
+			Number of solver position iterations. The greater the number of iterations, the more accurate the simulation will be, at the cost of CPU performance.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/sleep_time_threshold" type="float" setter="" getter="" default="0.5">
+			Time in seconds a [RigidBody3D] will spend below the sleep velocity threshold before going to sleep.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/sleep_velocity_threshold" type="float" setter="" getter="" default="0.03">
+			Velocity of certain points on the bounding box of a [RigidBody3D] below which it can be put to sleep.
```

Comment:
It's going to be difficult to describe this in a brief but exact way, but they are basically:
- The center of mass
- The centers of the 2 faces of the bounding box that are the furthest away from the center of mass
These are used to approximate both the movement and rotation of the object. If all points move less than this (linear) velocity (m/s), the body is candidate for sleeping.

Reply:
Are the exact points necessarily relevant? Would it be fine to just have it be something like this?

```suggestion
			Velocity of certain points on the bounding box of a [RigidBody3D] below which it can be put to sleep. These points are used to approximate both the movement and rotation of the [RigidBody3D].
```